src/async_impl/client.rs

```diff
@@ -471,6 +478,22 @@ impl ClientBuilder {
         };
         self
     }
+
+    /// Sets the `SETTINGS_INITIAL_WINDOW_SIZE` option for HTTP2 stream-level flow control.
+    ///
+    /// Default is 65,535
```

Comment:
I'd change this to state that the default may change internally to satisfy the common cases.